### PR TITLE
Add osx support to install via homebrew.

### PR DESCRIPTION
**Pull request by pr0d1r2**
See: https://github.com/pr0d1r2/homebrew-contrib/blob/master/youtube-dl-gui.rb

### CODE REVIEW

**Comment by pr0d1r2** [comment]
Added python dependency.
File `empty.lproj` is required to consider bundle as application.
Translations not yet handled (only needed functionality for now).
Moved `osx` folder outside package dir.

**Comment by MrS0m30n3** [comment]
It's not hard to handle the translation files.

Currently youtube-dlg searches this three directories:
 (\__main\__ dir, library dir, /usr/share/youtube-dlg/locale).
 
So you basically need to edit the **set_locale_files** method in the setup.py

```python
if PY2EXE:
    dst = os.path.join(PY2EXE_LOCALE_DIR, locale_lang)
elif os.name == 'nt':
    dst = os.path.join(WIN_LOCALE_DIR, locale_lang)
+ elif sys.platform == 'darwin':
+   dst = os.path.join( YOUR DESTINATION PATH HERE , locale_lang)
else:
    dst = os.path.join(LINUX_LOCALE_DIR, locale_lang)
```

And you also need to add the following line in your setup.py

```python
elif sys.platform == 'darwin':
    create_scripts()
+   set_locale_files(data_files)
    params = {'data_files': data_files, 'scripts': ['build/_scripts/youtube-dl-gui']}
```

If you want to add a new directory for the translations files you should also add the directory at the 
**end** of SEARCH_DIRS list, else youtube-dlg won't be able to find them.

See: https://github.com/MrS0m30n3/youtube-dl-gui/blob/master/youtube_dl_gui/utils.py#L191

**Comment by pr0d1r2** [comment]
Changes done. Confirmed that locales work on my machine :)

Thank you for quick guide on writing python packages.

File: youtube_dl_gui/utils.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import subprocess
 
 from .info import __appname__
+from youtube_dl_gui import __version__
```
Comment (MrS0m30n3) [inline youtube_dl_gui/utils.py]: You should import the version directly from the version.py file.

```Python
from .version import __version__
```

**Comment by pr0d1r2** [comment]
Done :)

**Comment by MrS0m30n3** [comment]
Thank you @pr0d1r2

**Comment by pr0d1r2** [comment]
It was a pleasure :smile:

Homebrew package updated with this repo: https://github.com/pr0d1r2/homebrew-contrib/commit/4d94c865a8eafed79db8d6dd74facbc6754a247a

**Pull request merged**